### ISSUE TITLE
Feature Request: Ability to show total duration

### ISSUE THREAD

**Issue by Californian** [rianadon/timer-bar-card#125]
Hello,

Love this card, very useful for glanceable timer display!

I have one timer whose length varies quite a bit at different times, and as such it would be very helpful if I could see the current duration in the card. E.g. if there are 3 minutes 20 seconds remaining on a 5 minute timer, it would show `03:20 / 05:00`.

Happy to contribute if this is a feature that would be accepted!

**Comment by rianadon** [comment]
Yes! I haven't had much time to work on the backlog of issues I assigned to myself, so I would be absolutely emphatically overjoyed if you contributed this.
